client/client_test.go

```diff
@@ -270,80 +260,6 @@ func TestAddListTarget(t *testing.T) {
 	assert.True(t, newFileFound, "second changelist file not found")
 
 	changelistDir.Close()
-
```

Comment:
It shouldn't have been. Git did a weird merge on this file and I just realized I pulled this file from my forked master rather than docker/master. Fixing now.